pkg/internalapis/featurestates/config/cns.vmware.com_cnscsisvfeaturestates.yaml

```diff
@@ -0,0 +1,68 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: cnscsisvfeaturestates.cns.vmware.com
+spec:
+  group: cns.vmware.com
+  names:
+    kind: CnsCsiSvFeatureStates
+    listKind: CnsCsiSvFeatureStatesList
+    plural: cnscsisvfeaturestates
+    singular: cnscsisvfeaturestate
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: CnsCsiSvFeatureStates is the Schema for the cnscsisvfeaturestates
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: CnsCsiSvFeatureStatesSpec defines the desired state of CnsCsiSvFeatureStates
+            properties:
+              featureStates:
+                items:
+                  description: FeatureState defines the feature name and its state
+                  properties:
+                    enabled:
+                      description: Enabled is set to true when feature is enabled,
+                        else it is set to false
+                      type: boolean
+                    name:
+                      description: Name is the unique identifier of the feature, this
+                        must be unique
+                      type: string
+                  required:
+                  - enabled
+                  - name
+                  type: object
+                type: array
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
```

Comment:
With status added as subResource, it is preventing operations like modify/update/add on status field. Please remove the subresources section